tool/bindings_generator/generate_bindings.dart

```diff
@@ -8,22 +8,21 @@ import 'translator.dart';
 import 'util.dart';
 import 'webref_idl_api.dart';
 
+/// Returns the part of the header common to every Dart file generated from the
+/// IDL.
 String generateHeader() {
-  return '''// Copyright (c) $year, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
+  return '''$licenseHeader
 
+// ignore_for_file: unused_import
```

Comment:
This is unfortunate, but to do better we'd have to do fine grained dependency tracking because the IDL seems to assume a global namespace. I'm willing to do this work, but it doesn't seem very valuable.

Reply:
Maybe I'll do it in my free time 😄

Reply:
This is done now, except for `package:js/js.dart hide JS`, there is a TODO in the code.